src/Compilers/VisualBasic/Test/Emit/PDB/PDBConstLocalTests.vb

```diff
@@ -169,51 +169,35 @@ End Class
                 source,
                 TestOptions.DebugDll)
 
-            compilation.VerifyDiagnostics(
-                 Diagnostic(ERRID.WRN_UnusedLocalConst, "o").WithArguments("o"),
-                 Diagnostic(ERRID.WRN_UnusedLocalConst, "s").WithArguments("s"),
-                 Diagnostic(ERRID.WRN_UnusedLocalConst, "f").WithArguments("f"),
-                 Diagnostic(ERRID.WRN_UnusedLocalConst, "d").WithArguments("d"),
-                 Diagnostic(ERRID.WRN_UnusedLocalConst, "dec").WithArguments("dec"),
-                 Diagnostic(ERRID.WRN_UnusedLocalConst, "dt").WithArguments("dt"))
-
-            Dim actual As XElement = GetPdbXml(compilation, "C.M")
-            Dim invariantStr = actual.ToString()
-            invariantStr = invariantStr.Replace("-3,402823E+38", "-3.402823E+38")
-            invariantStr = invariantStr.Replace("1,79769313486232E+308", "1.79769313486232E+308")
-            invariantStr = invariantStr.Replace("2,98187743664301E-266", "2.98187743664301E-266")
-            invariantStr = invariantStr.Replace("value=""1,5""", "value=""1.5""")
-            Dim expected = <symbols>
```

Comment:
Why do the values have comma decimal separator? Isn't the reader using InvariantCulture?